src/app/components/login/login.component.html

```diff
@@ -31,4 +31,5 @@
       <button mat-raised-button color="secondary" [routerLink]="['/register']">Register</button>
     </mat-card-actions>
   </mat-card>
+  <a (click)="openDialog()" class="reset-password-link">Glemt kodeord? Nulstil det her!</a>
```

Comment:
1. Hold dig til engelsk som resten af login formen.

2. Hvad med noget design ala det her? https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2Fimage.shutterstock.com%2Fz%2Fstock-vector-ui-elements-login-registration-and-forgot-password-form-flat-design-261288158.jpg&f=1